### PR TITLE
[WIP] In-page nav for About

### LINKED ISSUE #1835: About-page content needs "back to top" links, and may benefit from in-page nav
@toolness noted this problem long ago, in #1123

A right-nav topical list might be handy, too.

### PR DESCRIPTION
**Pull request by ericronne**
Closes #1835 

Kernel of a notion, pairing required to determine overall structure &c

### CODE REVIEW

File: data_explorer/templates/about.html [hunk 1]
```diff
@@ -87,8 +72,18 @@ <h2 id="schedules">What are these schedules?</h2>
       <p>IT Schedule 70 offers federal, state and local governments innovative solutions to their information technology needs.<br />
       <i>Note: IT Schedule 70 is newly added so available results in CALC is currently limited.</i></p>
     </div><!--content-->
+
+    {% block sidebar %}
+    <div id="right-nav" class="sidenav-in-page-layout__nav columns three">
+      <h4>Topics</h4>
+      <ul>
+        <li><a href="#schedules">What are these schedules of which you speak?</a>
```
Comment (tbaxter-18f) [inline data_explorer/templates/about.html]: WIP, I know, but you're missing the closing `li` and `ul` here.

File: frontend/source/sass/components/_page.scss [hunk 2]
```diff
@@ -92,7 +112,7 @@ footer {
   border-bottom: $space-half solid $background-color;
 
   /* This will only apply to cards that do not have a .card__footer.
-   * That class applies these styles as a top border to approximate the same thing. */
+  * That class applies these styles as a top border to approximate the same thing. */
```
Comment (hbillings) [inline frontend/source/sass/components/_page.scss]: plz don't kill the spaces; it's for aligning the text <3

Comment (ericronne) [reply frontend/source/sass/components/_page.scss]: dam linter. linter-culpa.

**Comment by hbillings** [comment]
Is this iceboxed?

**Comment by ericronne** [comment]
sure. not a priority.

**Comment by ericronne** [comment]
OBE

**Pull request closed without merging**